src/logic/Preferences.swift

```diff
@@ -104,8 +104,8 @@ class Preferences {
     // not exposed as preferences now but may be in the future, probably through macro preferences
     static var fontColor: NSColor { .white }
     static var windowPadding: CGFloat { 18 }
-    static var interCellPadding: CGFloat { 5 }
-    static var intraCellPadding: CGFloat { 5 }
+    static var interCellPadding: CGFloat { 15 }
+    static var intraCellPadding: CGFloat { 8 }
```

Comment:
I'm not sure to understand this change. Is it related to the goal of the MR?

Reply:
This fix is unrelated to the changes here. The modification was made because many AltTab users around me have reported that the spacing between thumbnails is too small, so I changed it to the thumbnail spacing of Windows 11. If you disagree, we can leave this part unchanged for now and consider revising it later as a separate advanced configuration option.

Reply:
![83ab94fdb3b446c1e98e5f63bc2f60ef](https://github.com/lwouis/alt-tab-macos/assets/5805484/846ac140-5056-4fab-98f8-b7cbafb3b534)

Changed the thumbnail spacing equals to Windows 11 spacing.

Reply:
I think changes shouldn't be bundled. It can create confusion while reviewing MRs. If you propose to make multiple unrelated changes, I suggest opening multiple MRs, each with their goal. They can be reviewed with more clarity.

Furthermore, we use a convention on commits. You can learn [a bit more here](https://alt-tab-macos.netlify.app/contributing#raising-a-pull-request). One commit = 1 change. Here there should be a `fix:` for the traffic light opacity issue, since it's indeed a bug fix. Then there should be a `feat:` to update the spacings, since it's not a bug fix bug an improvement. Finally, for documentation, one could use `docs:` or `refactor:` for the commit which documents the existing code.

Regarding the change of spacing: I think Windows 11 Spacings should live in a Windows 11 theme. We already support Themes in Preferences > Appearance. Let's add a Windows 11 theme. Other people were asking about it I believe.